mail_tracking/models/mail_message.py

```diff
@@ -86,6 +88,16 @@ def _get_error_description(self, tracking):
 
     def tracking_status(self):
         """Generates a complete status tracking of the messages by partner"""
+        catchall_domain = self.env['ir.config_parameter'].sudo().get_param(
+            'mail.catchall.domain')
+        db_aliases = {}
+        if catchall_domain:
+            db_aliases = set(
+                "%s@%s" % (rec['alias_name'], catchall_domain)
+                for rec in self.env['mail.alias'].search_read([
```

Comment:
As said, can we cached (through a method decorated with `orm_cache` in alias model) this alias list and invalidate this cache (I think it's `self.cache.clear()` in the corresponding model) when creating/modifying one alias? This search is still hammering a lot as it's done for each record.